sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RemoteDependencyData.cs

```diff
@@ -52,6 +52,13 @@ public RemoteDependencyData(int version, Activity activity, ref ActivityTagsProc
                     Data = depDataAndType[0]?.ToString().Truncate(SchemaConstants.RemoteDependencyData_Data_MaxLength);
                     Target = activityTagsProcessor.MappedTags.GetDbDependencyTarget().Truncate(SchemaConstants.RemoteDependencyData_Target_MaxLength);
                     Type = s_sqlDbs.Contains(depDataAndType[1]?.ToString()) ? "SQL" : depDataAndType[1]?.ToString().Truncate(SchemaConstants.RemoteDependencyData_Type_MaxLength);
+
+                    // special case for db.name
+                    var dbName = AzMonList.GetTagValue(ref activityTagsProcessor.MappedTags, SemanticConventions.AttributeDbName)?.ToString().Truncate(SchemaConstants.KVP_MaxValueLength);
+                    if (dbName != null)
+                    {
+                        Properties.Add(SemanticConventions.AttributeDbName, dbName);
```

Comment:
Properties is initialized at later point in the same method, does this get added?

Reply:
It's better to move this logic to `GetDbDependencyTarget`, so we don't read from list twice.

Reply:
Some issue with my local, test was passing so did not catch this.

Updated `GetDbDependencyTarget`